packages/cli/src/lib/defaults/deploy-modules/ens/index.ts

```diff
@@ -49,11 +52,22 @@ class ENSPublisher implements DeployModule {
       ? new Wallet(config.privateKey).connect(connectionProvider)
       : undefined;
 
-    const client = new PolywrapClient({
-      packages: [
-        {
-          uri: defaultInterfaces.ethereumProvider,
-          package: ethereumProviderPlugin({
+    const clientConfig = new ClientConfigBuilder()
+      .addDefaults()
+      .add({
+        envs: {
```

Comment:
The redirects and ipfs env shouldn't be needed here. The ipfs resolver has retries set in the default config and the redirects won't do anything anymore since the ethereum wrapper was merged. Same for the other ens deploy module.

Reply:
Removed them. Let's let the test turn green and we're good!